src/shared/types/abTests/epic.ts

```diff
@@ -112,7 +112,8 @@ export interface EpicTest extends EpicTestFromTool {
 export const epicTestFromToolSchema = testSchema.extend({
     name: z.string(),
     status: testStatusSchema,
-    locations: z.array(countryGroupIdSchema),
+    locations: z.array(countryGroupIdSchema), //deprecated
```

Comment:
shall we make `locations` optional as well now? Presumably SAC will stop setting it at the same time that it starts setting `regionTargeting`